test/Microsoft.IdentityModel.Tests/IdentityUtilities.cs

```diff
@@ -93,6 +94,99 @@ public static JwtSecurityToken CreateJwtSecurityToken(string issuer, string audi
             return new JwtSecurityToken(header, payload, header.Base64UrlEncode(), payload.Base64UrlEncode(), "" );
         }
 
+        public static string CreateSamlSecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
+        {
+            SecurityToken securityToken = tokenHandler.CreateToken(securityTokenDescriptor);
+            StringBuilder sb = new StringBuilder();
+            XmlWriter writer = XmlWriter.Create(sb);
+            tokenHandler.WriteToken(writer, securityToken);
+            writer.Flush();
+            writer.Close();
+            return sb.ToString();
+        }
+
+        public static string CreateSaml2SecurityToken(SecurityTokenDescriptor securityTokenDescriptor, SecurityTokenHandler tokenHandler)
+        {
+            SecurityToken securityToken = tokenHandler.CreateToken(securityTokenDescriptor);
```

Comment:
you could write this as:
var token = tokenHandler.CreateToken(securityTokenDescriptor)
return tokenHandler.WriteToken(token);

OR

return tokenHandler.WriteToken(tokenHandler.CreateToken(securityTokenDescriptor);

Reply:
Fixed